components/style/values/specified/basic_shape.rs

```diff
@@ -528,3 +629,77 @@ impl ToCss for FillRule {
         }
     }
 }
+
+/// https://drafts.fxtf.org/css-masking-1/#typedef-geometry-box
+#[derive(Clone, PartialEq, Copy, Debug)]
+#[cfg_attr(feature = "servo", derive(HeapSizeOf))]
+pub enum GeometryBox {
+    Fill,
+    Stroke,
+    View,
+    Shape(ShapeBox),
```

Comment:
Would it be bad to call this enum value "ShapeBox" too?  "Shape" to me sounds like it's going to have a shape inside it, but actually it has the geometry boxes which can be used in conjunction with basic shapes.